lib/modules/subredditManager.js

```diff
@@ -1072,24 +1074,50 @@ async function getSubreddits() {
 	} while (after);
 
 	// Remove duplicate subreddits
-	mySubreddits = _.uniqBy(mySubreddits, sr => sr.display_name);
+	mySubreddits = organizeMyData(mySubreddits, sr => sr.display_name);
 
-	mySubreddits.sort((a, b) => {
-		const adisp = a.display_name.toLowerCase();
-		const bdisp = b.display_name.toLowerCase();
+	gettingSubreddits = false;
+	populateSubredditDropdown();
+}
+
+let myMultis = [];
+async function getMultis() {
+	if (gettingMultis) return;
+	gettingMultis = true;
+
+	myMultis = [];
+	const multis = await ajax({
```

Comment:
RES would need to monitor the two multi creation UI elements (frontpage sidebar and subscribe button popup)  and bust the cache manually. definitely worth doing but imo we could defer that to a separate PR.